.gitignore

```diff
@@ -2,4 +2,7 @@ node_modules/
 tests/
 .nvmrc
 .prettierrc.js
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+/.settings/
```

Comment:
did not know how to prevent this update, it's really just some temp files my IDE copies into all projects I work on.